src/main/java/taxi/controller/authentication/LogoutController.java

```diff
@@ -0,0 +1,16 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getSession().invalidate();
+        resp.sendRedirect("/index");
```

Comment:
After logging out user hasn't access to `index` page
```suggestion
        resp.sendRedirect("/login");
```